### ISSUE TITLE
Room Editor: Show the current grid X/Y values on the status bar, in addition to the mouse position

### ISSUE THREAD

**Issue by gnysek** [YoYoGames/GameMaker-Bugs#2851]
### Is your feature request related to a problem?

As assets in Room Editor can be aligned to grid, sometimes it's good to know x,y of grid cell, to create a code which spawns something in room, instead of placing instance in room editor.

However, room editor only shows current x,y position. It would be nice, that it would also show x,y of current grid cell when such is hovered.

An example of how that could look:

![obraz](https://github.com/YoYoGames/GameMaker-Feature-Requests/assets/524094/4db66de4-8b3f-4aa6-8203-28a573bc607c)

### Describe the solution you'd like

Expand this part of room editor:
![obraz](https://github.com/YoYoGames/GameMaker-Feature-Requests/assets/524094/764630c2-e4f4-4134-85a6-6d6a0e05b39b)
So it displays hovered grid x,y, according to current grid settings:
![obraz](https://github.com/YoYoGames/GameMaker-Feature-Requests/assets/524094/c30a52de-c1ce-4f6b-a3bd-5c3f5e1f5e16)

Final result, when hovering cell 1,1:
![obraz](https://github.com/YoYoGames/GameMaker-Feature-Requests/assets/524094/4db66de4-8b3f-4aa6-8203-28a573bc607c)

![Image](https://github.com/user-attachments/assets/a2820eee-acf3-4873-a4e2-01868c609f4b)

### Additional context

Such feature was present in Delphi era versions of GM from what I remember (or it's Mandela effect).

Another great feature somehow connected to this FR, is also showing tile index (and row, column too): https://github.com/YoYoGames/GameMaker-Bugs/issues/3565

**Comment by scott-dunbar** [comment]
Yeah, issue very much still exists as shown in my comment in  IDE v2024.1400.2.937  Runtime v2024.1400.2.922.

Between the merge and my previous test there had been 3 builds out of develop, so im not sure if the Github tagging system is reliable or im not sure why it wouldnt have been present when the video clearly shows the addition of the second set of the coordinates that previously did not exist.
